s/elements/stickpad/element.ts

```diff
@@ -0,0 +1,88 @@
+import {html} from "lit"
+import {MagicElement, mixinCss} from "@chasemoskal/magical"
+import * as v2 from "../../tools/v2.js"
+import {Nub} from "../../types.js"
+
+import styles from "./style.css.js"
+import {property} from "lit/decorators.js"
+import {StickpadStarters} from "./types.js"
+import {StickStarters} from "../stick/types.js"
+import {NubInputEvent} from "../../events/input.js"
+import {prepBaseEvents} from "../../setups/prep-base-events.js"
+import {prepDomControls} from "../../setups/prep-dom-controls.js"
+import {setupBaseEvents} from "../../setups/setup-base-events.js"
+import {setupStickpadEvents} from "./setups/setup-stickpad-events.js"
+import {setupWindowEvents} from "../../setups/setup-window-events.js"
+
+@mixinCss(styles)
+export class NubStickpad extends MagicElement {
+
+	@property({type: String, reflect: true})
+	name: string = "1"
+
+	get nubStickBasePart() {
+		return this.shadowRoot?.querySelector("nub-stick-graphic")!
+			.shadowRoot?.querySelector("[part='base']")! as HTMLElement
+	}
+
+	get nubStickPart() {
+		return this.shadowRoot?.querySelector("nub-stick-graphic")!
+			.shadowRoot?.querySelector("[part='stick']")! as HTMLElement
+	}
```

Comment:
we need to fix this bad shadow-boundary drilling.

the purpose of the shadow dom, is that it provide a cloak/shield that is meant to prevent outer elements from meddling with inner elements -- drilling through the shadow dom to defeat that cloak/shield is counter-productive, and is therefore a bad code smell (to understand this better, look up the software principle of "encapsulation").